src/utils/executeAsTask.ts

```diff
@@ -0,0 +1,29 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import { IActionContext } from 'vscode-azureextensionui';
+import { addDockerSettingsToEnv } from './addDockerSettingsToEnv';
+
+export async function executeAsTask(context: IActionContext, command: string, name: string, addDockerEnv: boolean, workspaceFolder?: vscode.WorkspaceFolder, cwd?: string): Promise<vscode.TaskExecution> {

```

Comment:
Nit: just use an Enum, or string literal type for `addDockerEnv` parameter. Then you don't need to put a comment everywhere to remind ourselves what the `true` or `false` value stands for.